File: week-02/day-1/src/DoubleItems.java
import java.util.Arrays;

public class DoubleItems {

  public static void main(String[] args) {
    // Create an array variable named `numList`
    //   with the following content: `[3, 4, 5, 6, 7]`

    int[] numList = {3, 4, 5, 6, 7};
    System.out.println(Arrays.toString(numList));

    // Double all the values in the array
    for (int i = 0; i < numList.length; i++) {
      numList[i] *= 2;
    }
    System.out.println(Arrays.toString(numList));
  }
}

File: python_solutions/week-02/day-1/arrays/double_items.py
# - Create an array variable named `numList`
#   with the following content: `[3, 4, 5, 6, 7]`
# - Double all the values in the array

num_list = [3, 4, 5, 6, 7]
print(num_list)

num_list = list(map(lambda x: x * 2, num_list))
print(num_list)
